crates/experimental-api/src/gen.rs

```diff
@@ -9,10 +9,20 @@ pub mod gen {
     use serde_json::Value;
 
     use iamgroot::jsonrpc;
+    use starknet_crypto::{pedersen_hash, poseidon_hash_many, FieldElement};
```

Comment:
In general `gen.rs` contains only generated code and is not expected to be updater directly.

If something is missing, it is added to the codegen tool, so on each change of the spec painful merges can be avoided.

I will put this into README/CONTRIBUTING.

Reply:
Removed